internal/rest/avi_evh_obj.go

```diff
@@ -519,6 +520,7 @@ func (rest *RestOperations) AviVsChildEvhBuild(vs_meta *nodes.AviEvhVsNode, rest
 			Path: &path_match,
 		}
 		vHMatchRule := &avimodels.VHMatchRule{
+			Name:    &name,
```

Comment:
@arihantg : Is `NAME` unique key? In case of multiple MatchRule it might create problem.

Reply:
there will only be a single matchrule for a hostname.So we can keep the value hardcoded

Reply:
It will create problem for hostrule with GSLB FQDN or alias feature.

Reply:
@arihantg Have you tested the hostrule aliases and GSLB FQDN?

Reply:
I have tested this test  test_hostrule_fdqn_aliases in test_host_rules which is passing successfully.

Reply:
Thanks @arihantg